pkg/blobinfocache/internal/test/test.go

```diff
@@ -16,6 +16,10 @@ const (
 	digestCompressedA         = digest.Digest("sha256:3333333333333333333333333333333333333333333333333333333333333333")
 	digestCompressedB         = digest.Digest("sha256:4444444444444444444444444444444444444444444444444444444444444444")
 	digestCompressedUnrelated = digest.Digest("sha256:5555555555555555555555555555555555555555555555555555555555555555")
+	compressorNameU           = "compressorName/U"
+	compressorNameA           = "compressorName/A"
+	compressorNameB           = "compressorName/B"
+	compressorNameCU          = "compressorName/CU"
 )
 
 // GenericCache runs an implementation-independent set of tests, given a
```

Comment:
Absolutely non-blocking, it might be interesting to test _both_ the V2 implementation and the V1 implementation"
- call the V2 versions if available, which is “always” for the two users, or even just assert that V2 is available
- wrap the V2 implementation in a V1-only object to test that the V1 implementation, which is externally-observable, continues to work as expected for V1.

OTOH… “externally-observable” is, I fervently hope, such a theoretical concept, so who cares…